yarn.lock

```diff
@@ -9476,10 +9500,24 @@ terser-webpack-plugin@^1.1.0, terser-webpack-plugin@^1.4.1:
     webpack-sources "^1.4.0"
     worker-farm "^1.7.0"
 
-terser@^4.1.2:
-  version "4.3.0"
-  resolved "https://registry.yarnpkg.com/terser/-/terser-4.3.0.tgz#0259070576ff34d89c5cd2b8d91055cdce0212d5"
-  integrity sha512-w5CzrvQOwYAH54aG22IrUJI4yX1w62XQmMdEOM6H4w0ii6rc3HJ89fmcOGN5mRwBWfUgaqO7RJTp4aoY/uE+qQ==
+terser-webpack-plugin@^2.0.0:
+  version "2.0.0"
```

Comment:
There is already a patch release ([2.0.1](https://github.com/webpack-contrib/terser-webpack-plugin/releases/tag/v2.0.1))

Reply:
I re-created the yarn.lock to get the latest patch release.